test/auto/injectorSpec.js

```diff
@@ -2,6 +2,37 @@
 
 /* globals support: false */
 
+describe('injector.modules', function() {
+    it('should expose the loaded module info on the instance injector', function() {
+      var test1 = angular.module('test1', ['test2']).info({ version: '1.1' });
+      var test2 = angular.module('test2', []).info({ version: '1.2' });
+      module('test1');
+      inject(['$injector', function($injector) {
```

Comment:
Ah, yes you are right @gkalpak (as always) - I was thinking of this: https://github.com/angular/angular.js/issues/13542